lib/src/widgets/slidable.dart

```diff
@@ -323,15 +323,15 @@ class SlidableData extends InheritedWidget {
       (oldWidget.direction != direction);
 }
 
-/// A controller that keep tracks of the active [SlidableState] and close
+/// Creates a controller that keep tracks of the active [SlidableState] and close
 /// the previous one.
 class SlidableController {
-  /// Creates a controller that keep tracks of the active [SlidableState] and close
-  /// the previous one.
-  SlidableController({
-    this.onSlideAnimationChanged,
-    this.onSlideIsOpenChanged,
-  });
+  SlidableController(
+      {this.onSlideAnimationChanged,
+      this.onSlideIsOpenChanged,
+      this.onSliding,
```

Comment:
These callbacks should be added on Slidable and not the SlidableController. The SlidableController is used to keep tracks of the active one.